src/app.ts

```diff
@@ -140,17 +150,6 @@ export class CohortApp {
   }
 
   private async builDataSelector(btnGrp) {
-    // first set back button, in case data retrieval fails
-    select('nav a.navbar-brand.caleydo_app')
```

Comment:
I committed this to develop before, because I thought it doesnt work in my local environment.
But the issue was the clickhandler in tdp_core that i removed now.
I also moved this further up, because it doesn't have anything to do with the data selector anymore